src/error-handling/exercise.md

```diff
@@ -10,7 +10,7 @@ Rewrite `eval` to instead use idiomatic error handling to handle this error case
 and return an error when it occurs. We provide a simple `DivideByZeroError` type
 to use as the error type for `eval`.
 
-```rust,editable
```

Comment:
I'd prefer that we don't remove `editable` from the exercise sources, it can be useful to make small edits to them in place to demonstrate things to students.

What's the reason for removing these (both here and for the other exercises where you made the same change)? Are we just trying to prevent students from using the exercise slide to do the exercise (i.e. because if students navigate away from the slide they'll loose their changes)?

Reply:
Yeah, it's to prevent students from editing in-place, but your motivation is good.